### DISCUSSION TITLE
Move the changelog

### DISCUSSION THREAD

**Discussion by jenstornell** [vuejs/pinia#1038] [Ideas]
Now it's not very easy to find the changelog.

https://github.com/vuejs/pinia/blob/v2/packages/pinia/CHANGELOG.md

### Suggestions

1. Move it to the root of the project like most other projects.
2. Add a link to it in the README-file if there are reasons to not move it from the current location.

**Comment by posva** [comment]
This is how the generation works. I don't think it's much of an issue as you can find the link in releases and the documentation.
